### ISSUE TITLE
`puppeteer` become `dependencies`, not `devDependencies` in 2.5.0. it makes 300M zip file.

### ISSUE THREAD

**Issue by thomaszooman** [twilio/twilio-video.js#1036]
- [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**
```json
  "dependencies": {
    "@twilio/webrtc": "4.2.1",
    "backoff": "^2.5.0",
    "chromedriver": "2.28.0",
    "puppeteer": "^1.11.0",
    "ws": "^3.3.1",
    "xmlhttprequest": "^1.8.0"
  },
```
`puppeteer` become `dependencies`, not `devDependencies` in 2.5.0. it makes the code package more them 300M zip file because the Chromium.app. please confirm that does it REALLY need in runtime?

TODO

**Software versions:**

 - [ ] Browser(s):
 - [ ] Operating System:
 - [x] twilio-video.js:
 - [x] Third-party libraries (e.g., Angular, React, etc.):

**Comment by makarandp0** [comment]
Thank you for bringing up this issue @thomaszooman - This was not intentional - `puppeteer` is not needed at runtime. I have filed a bug to get this fixed (JSDK-2848)

Thanks
Makarand

**Comment by makarandp0** [comment]
@thomaszooman, Thank you again for reporting this issue - this is now fixed in [2.5.1](https://github.com/twilio/twilio-video.js/releases/tag/2.5.1).
